clients/cobol-lsp-vscode-extension/src/services/ZoweApi.ts

```diff
@@ -41,28 +42,39 @@ export class ZoweApi {
     }
 
     public async fetchMember(dataset: string, member: string, profileName: string): Promise<string> {
-        const session: Session = await this.createSession(profileName);
+        try {
+            const session: Session = await this.createSession(profileName);
 
-        // default should be https
-        const rpath = `${ZOSMF_PREFIX}${dataset}(${member})`;
-        return await RestClient.getExpectString(session, rpath, [{
-            "Content-Type": "application/json", "X-CSRF-ZOSMF-HEADER": "",
-        }]);
+            // default should be https
+            const rpath = `${ZOSMF_PREFIX}${dataset}(${member})`;
+            return await RestClient.getExpectString(session, rpath, [{
+                "Content-Type": "application/json", "X-CSRF-ZOSMF-HEADER": "",
+            }]);
+        } catch (error) {
+            throw convertError(error);
+        }
     }
 
     public async listMembers(dataset: string, profileName: string): Promise<string[]> {
-        // default should be https
-        const session: Session = await this.createSession(profileName);
-        const rpath = `${ZOSMF_PREFIX}${dataset}/member`;
-        const result = await RestClient.getExpectJSON(session, rpath, [{
-            "Content-Type": "application/json", "X-CSRF-ZOSMF-HEADER": "",
-        }]);
-        // tslint:disable-next-line: no-string-literal
-        return result["items"].map((i: any) => i.member);
+        try {
+            // default should be https
+            const session: Session = await this.createSession(profileName);
+            const rpath = `${ZOSMF_PREFIX}${dataset}/member`;
+            const result = await RestClient.getExpectJSON(session, rpath, [{
+                "Content-Type": "application/json", "X-CSRF-ZOSMF-HEADER": "",
+            }]);
+            // tslint:disable-next-line: no-string-literal
+            return result["items"].map((i: any) => i.member);
+        } catch (error) {
+            throw convertError(error);
+        }
     }
 
     public async createSession(profileName: string) {
         const profile = (await new BasicProfileManager(this.createProfileParams()).load({ name: profileName })).profile;
+        if (profile.password === "") {
```

Comment:
Can profile be undefined here?

Reply:
Good point! I think that it is possible in case when the user deletes the profile on filesystem right before downloading copybooks.
We need another Error type and Error message for this case. I'll add them.

Reply:
Ha! I can't reproduce it.
When I choose the profile, then delete the file and try to download copybook via "quick fix", the IDE asks me to choose the other profile.
When I delete all profiles, IDE shows an error "Zowe profile is missing".
This case already covered.